bibliopixel/drivers/SPI/errors.py

```diff
@@ -0,0 +1,35 @@
+PERMISSION_ERROR = """Cannot access SPI device.
+Please see
+
+    https://github.com/maniacallabs/bibliopixel/wiki/SPI-Setup
+
+for details.
+"""
+
+CANT_FIND_ERROR = """Cannot find SPI device.
+Please see
+
+    https://github.com/maniacallabs/bibliopixel/wiki/SPI-Setup
+
+for details.
+"""
+
+CANT_IMPORT_PERIPHERY_ERROR = """
```

Comment:
This one error should go into bp.project.importer.py with all the other importer errors.